source/reference/collection-level-filtering.txt

```diff
@@ -0,0 +1,203 @@
+.. _c2c-collection-filtering:
+
+==========================
+Collection Level Filtering
+==========================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. versionadded:: 1.1
+
+{+c2c-product-name+} provides continuous data synchronization or a 
+one-time data migration between two MongoDB clusters. You can use
+collection level filtering to control the data that the :ref:`mongosync
+<c2c-mongosync>` utility transfers between the source and destination
+clusters.
+
+.. _c2c-configure-filter:
+
+Configure a Filter
+------------------
+
+.. important::
+
+   Once you start ``mongosync`` with a filter in place, the filter
+   cannot be modified directly. If you do need to modify the filter,
```

Comment:
I'd consider changing `If you do need to modify the filter` to `If you misconfigured a filter`. I don't want to give people the idea that filtered sync should currently be expected to be used in a mid migration filter changing manner